digitalocean/README.md

```diff
@@ -9,7 +9,8 @@ This is an simple provisioner for DigitalOcean [Block Storage](https://www.digit
    Create the secret: `kubectl create -f manifests/digitalocean-secret.yaml`
 2. Deploy the provisioner: `kubectl create -f manifests/digitalocean-provisioner.yaml`
 3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml)  
-   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`
+   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`.
+   - Please ensure the flexvolume plugin directory path is accessible in `kube-controller-manager` by adding a hostPath mount to the manifest file on the master node (`/etc/kubernetes/manifests/kube-controller-manager.yaml`), using [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml) as an example.
```

Comment:
I'm wondering if we could rephrase the sentence in a better way. Something like: (maybe, I'm not good at english at al)

> **Note:** Please ensure that `kube-controller-manager` has access to the flexvolume plugin directory. If running `kube-controller-manager` as a pod, please take a look at [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml), for how this can be done.

Reply:
We should probably also document the required `bootkube` tweak I use, as documented in https://github.com/kubernetes-incubator/external-storage/issues/531. Then we could close that issue after merging.